src/main/app/src/app/zoeken/zoek/zoek.component.ts

```diff
@@ -79,6 +81,9 @@ export class ZoekComponent implements AfterViewInit {
             this.paginator.length = data.totaal;
             this.hasSearched = true;
             this.zoekResultaat = data;
+            this.hasZaken = this.zoekResultaat.filters.TYPE.find(f => f.naam === 'ZAAK')?.aantal > 0;
```

Comment:
Dit vraagt diepere Typescript kennis ... Zou je dit eens willen toelichten tijdens een stand-up? Maar kun je hier in plaats van de strings ZAAK, TAAK en DOCUMENTEN niet de enum gebruiken?

Reply:
Ja, Ik kan dit wel eens toelichten. heb er en enum van gemaakt.